src/components/buttons/ToggleButton.vue

```diff
@@ -23,20 +27,40 @@
       },
       label: String,
       tooltipOn: String,
-      tooltipOff: String
+      tooltipOff: String,
+      color: String
     },
+    data: () => ({
+      opacity: false
+    }),
 
     mounted: function () {
       if (this.label && !this.id) {
         throw "You need to define aui-toggle-button id attribute to properly match label to the toggle"
       }
       this.$refs.toggle.addEventListener('change', this.emitChange);
       this.$nextTick(() => this.$refs.toggle.busy = this.busy);
+
+      if (this.color) {
+        this.changeColor();
+      }
     },
 
     methods: {
+      changeColor() {
```

Comment:
I believe this should be replacable by CSS too? You have already found an attribute `enabled` that determines when the color should be applied.